### DISCUSSION TITLE
how to edit docker-compose.yml?

### DISCUSSION THREAD

**Discussion by dariotal** [teslamate-org/teslamate#2761] [Q&A]
hi, i have updated to the latest teslamate version. 
I entered the line "- ENCRYPTION_KEY =" with my password, but now I can't save.
it tells me that "docker-compose.yml" is not overwritable. how can I do?
thanks, dario

**Comment by LelandSindt** [comment]
@dariotal sounds like a file system permissions issue.. Are you the file owner, and do you have sufficient rights to modify?

**Reply by natrlhy** [reply]
You may have to run `sudo vi docker-compose.yml` if you are logged in with a non-root account
